### DISCUSSION TITLE
Finding a particular trait in vscode

### DISCUSSION THREAD

**Discussion by dodalovicgran** [rust-lang/rust-analyzer#15447] [Q&A]
Hi, I'm trying to search for types in vscode, by invoking `ctrl+t` and typing something like `Display` or `Debug`, but nothing comes back. 

How can I find something coming from rust in vscode?

Thank You

**Comment by Veykril** [comment]
There was a bug specifically regarding traits in the symbol search, next release has this fixed

**Reply by dodalovicgran** [reply]
Nice, thank you!
